recipes/cmpitool/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "cmpitool" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cmpitool-{{ version }}.tar.gz
+  sha256: c1b4eeebda3fb75c59ef59b22ba9dfda68a43fd7a67f8456be61d2d05b6e6bc9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - geopandas
+    - matplotlib-base
+    - numpy
+    - pandas >=1.0.0
+    - pooch
+    - regionmask
+    - seaborn
+    - tqdm
+    - xarray
+
+test:
+  imports:
+    - cmpitool
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/JanStreffing/cmpitool
+  summary: cmpitool
+  license: GPL-2.0-or-later
+  license_file: LICENSE
+
+
```

Comment:
This is superseded by the content below.
```suggestion
```

Reply:
removed